queries/elixir/context.scm

```diff
@@ -0,0 +1,18 @@
+(binary_operator
+  left: (_)
+  right: (_) @context)
+
+(pair
+  key: (_)
+  value: (_) @context)
+
+((unary_operator
+   operand: (call
+      target: (identifier)
+      (arguments (_)))) @_op (#lua-match? @_op "@[%w_]+")) @context
+
+(stab_clause right: (body (_) @context.end)) @context
+
+(call
+  target: (identifier) @_ident
```

Comment:
You don't need the `@_ident` capture.

Reply:
Thanks, leftover from an earlier iteration of the query.